content/sensu-go/6.8/catalog/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Sensu Catalog"
+description: "PLACEHOLDER."
+product: "Sensu Go"
+version: "6.8"
+weight: 55
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.8:
+    identifier: catalog
+---
+
+{{% notice commercial %}}
+**COMMERCIAL FEATURE**: Access the Catalog API in the packaged Sensu Go distribution.
+For more information, read [Get started with commercial features](../../commercial/).
+{{% /notice %}}
+
+{{% notice note %}}
+**NOTE**: The Sensu Catalog is in public preview and is subject to change.
+{{% /notice %}}
+
+The Sensu Catalog...
+
+**PLACEHOLDER**
+
```

Comment:
I'm not sure we'll keep the catalog docs all together in a single category. If we do, I will add content to this page.